sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/models/AnalyzeDocumentOptions.java

```diff
@@ -81,4 +82,16 @@ public AnalyzeDocumentOptions setDocumentAnalysisFeatures(List<DocumentAnalysisF
         this.documentAnalysisFeatures = documentAnalysisFeatures;
         return this;
     }
```

Comment:
Should we remove this method since we add the new method?`setDocumentAnalysisFeatures(DocumentAnalysisFeature... documentAnalysisFeatures)`

And the method is still in beta

Reply:
Planning on keeping both.